artemis-protocols/artemis-openwire-protocol/src/test/java/org/apache/activemq/artemis/core/protocol/openwire/OpenWireMessageConverterTest.java

```diff
@@ -91,4 +92,20 @@ public void createMessageDispatch() throws Exception {
       }
 
    }
+
+   @Test
+   public void testBytesPropertyConversionToString() throws Exception {
+      final String bytesPropertyKey = "bytesProperty";
+
+      ICoreMessage coreMessage = new CoreMessage().initBuffer(8);
+      coreMessage.putBytesProperty(bytesPropertyKey, "TEST".getBytes());
+
+      MessageReference messageReference = new MessageReferenceImpl(coreMessage, Mockito.mock(Queue.class));
+      AMQConsumer amqConsumer = Mockito.mock(AMQConsumer.class);
+      Mockito.when(amqConsumer.getOpenwireDestination()).thenReturn(destination);
+
+      MessageDispatch messageDispatch = OpenWireMessageConverter.createMessageDispatch(messageReference, coreMessage, openWireFormat, amqConsumer, nodeUUID);
+
+      Assert.assertTrue(messageDispatch.getMessage().getProperty(bytesPropertyKey) instanceof String);
```

Comment:
Can you add a test where it's actually sending a message and converting to OpenWire and AMQP?

Reply:
I have just added the testBinaryPropertyConversionToString test converting from AMQP to OpenWire.